### PR TITLE
fix: Changes to aircraft.cfg to match the A32NX better

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes
Some minor changes to the aircraft.cfg and also fixing the atc_heavy config to 0 as the A320 is not considered heavy. 

Discord username (if different from GitHub): cdr_maverick

## Testing instructions
not required

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg [hunk 1]
```diff
@@ -86,11 +86,10 @@ ui_certified_ceiling = 39800 ; service ceiling / max certified operating altitud
 ui_max_range = 3500 ; max distance the aircraft can fly between take-off and landing in (NM)
 ui_autonomy = 7 ; max duration the aircraft can fly between take-off and landing in (Hrs)
 ui_fuel_burn_rate = 5300 ; average fuel consumption per hour (lbs/hr) - reminder: fuel density is ~6.7lbs per US gallon
-atc_id = "ASX320" ; tail number
-atc_id_enable = 0 ; enable tail number
-atc_airline = "" ; airline name
-atc_flight_number = "1123" ; flight number
-atc_heavy = 1 ; heavy?
+atc_id = "FBW320" ; tail number
+atc_airline = "FBWSIM" ; airline name
```
Comment (tracernz) [inline fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg:90]: How does this sound in the sim TTS?

Comment (frankkopp) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg:90]: no idea :D :D - I don't use ingame ATC. Should we rather leave it?

Comment (2hwk) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg:90]: We should probably remove this and not define atc_id explicitly at the base level. We should leave it to livery to define atc_id (for example currently the house livery currently has atc_id: "A32NX" defined there)

Comment (frankkopp) [reply fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/aircraft.cfg:90]: in a sense this is the default livery so it should be defined I guess, right?

**Review by 2hwk — APPROVED** [review]

**Pull request merged**